nitric/proto/kvstore/v1/kvstore.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+package nitric.proto.kvstore.v1;
+
+import "google/protobuf/struct.proto";
+
+// protoc plugin options for code generation
+// option go_package = "nitric/v1;v1";
+option go_package = "github.com/nitrictech/nitric/core/pkg/proto/kvstore/v1;kvstorepb";
+option java_package = "io.nitric.proto.kvstore.v1";
+option java_multiple_files = true;
+option java_outer_classname = "KvStore";
+option php_namespace = "Nitric\\Proto\\KvStore\\V1";
+option csharp_namespace = "Nitric.Proto.KvStore.v1";
+
+// Service for storage and retrieval of simple JSON keyValue
+service KvStore {
+  // Get an existing value
+  rpc GetKey (KvStoreGetRequest) returns (KvStoreGetResponse);
```

Comment:
@jyecusch should we update these to prefix the method name to the messages?
e.g. `KvStoreGetKeysRequest`